resources/prosody-plugins/mod_muc_limit_messages.lua

```diff
@@ -0,0 +1,98 @@
+-- A module to limit the number of messages in a meeting
+-- Needs to be activated under the muc component where the limit needs to be applied
+-- Copyright (C) 2023-present 8x8, Inc.
+
+local id = require 'util.id';
+local st = require 'util.stanza';
+
+local get_room_by_name_and_subdomain = module:require 'util'.get_room_by_name_and_subdomain;
+
+local messages_per_room;
+local drop_limits_authenticated;
+
+local function load_config()
+    messages_per_room = module:get_option_number('muc_limit_messages_count');
+    drop_limits_authenticated = module:get_option_boolean('muc_limit_messages_check_token', false);
```

Comment:
nit: Why such different names for the config options and the local variables? Why not call them `count` and `check_token` if that's what they do?